authentication-typescript/template.json

```diff
@@ -1,12 +1,15 @@
 {
     "package": {
         "dependencies": {
-            "@material-ui/core": "~4.11.0",
-            "@material-ui/icons": "^4.9.1",
+            "@mui/material": "^5.5.0",
+            "@mui/icons-material": "^5.5.0",
+            "@mui/styles": "^5.5.0",
+            "@emotion/react": "^11.8.2",
+            "@emotion/styled": "^11.8.1",
             "@brightlayer-ui/react-auth-workflow": "^2.4.0",
-            "@brightlayer-ui/react-themes": "^6.1.0",
-            "@brightlayer-ui/react-components": "^5.3.0",
-            "@brightlayer-ui/icons-mui": "^2.4.1",
+            "@brightlayer-ui/react-themes": "^6.1.1",
+            "@brightlayer-ui/react-components": "^5.4.0",
+            "@brightlayer-ui/icons-mui": "^2.8.1",
```

Comment:
You're still using Material-UI v4 based packages. Let's use the latest alpha @brightlayer-ui packages.